base_tier_validation/models/tier_validation.py

```diff
@@ -655,8 +655,10 @@ def request_validation(self):
                     if rec.evaluate_tier(td):
                         sequence += 1
                         vals_list.append(rec._prepare_tier_review_vals(td, sequence))
-                self._update_counter({"review_created": True})
         created_trs = tr_obj.create(vals_list)
+        self._invalidate_cache(["review_ids"])
+        if any(item.can_review for item in self):
```

Comment:
Let's do it a bit more elegant:

```suggestion
        if any(item.mapped("can_review")):
```